charts/flink-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Intalls the Spotify-version of the Flink-on-K8S-Operator
 name: flink-operator
-version: 0.0.1
```

Comment:
This is a patch version change (Semantic version is usually of this format: _<major>.<minor>.<patch>_). Changing the middle number would be better for this PR, unless this is an automatic system generated change.

Reply:
Format: MAJOR.MINOR.PATCH